lib/redmine_xlsx_format_issue_exporter/xlsx_export_helper.rb

```diff
@@ -98,7 +99,7 @@ def crlf_to_lf(value)
       value.is_a?(String) ? value.gsub(/\r\n?/, "\n") : value
     end
 
-    def write_item(worksheet, value, row_index, column_index, cell_format, is_id_column, id, hyperlink_format)
+    def write_item(worksheet, value, row_index, column_index, cell_format, is_id_column, id, hyperlink_format, float_format)
```

Comment:
I think not to need adding parameter **float_format** .
The caller judges cell format by value, and should passes it to **cell_format** .